examples/build.sbt

```diff
@@ -0,0 +1,22 @@
+name := "hyperspaceApp"
```

Comment:
Can you move the whole app to `examples/scala/...` so that we can put the C# example in `examples/csharp/...`?

Reply:
Done.